integration-tests/src/integration-tests/resources/ITSingleDomain_domain.properties

```diff
@@ -0,0 +1,17 @@
+#any property can be provided here from create-weblogic-domain-inputs.yaml
+#if a property is not defined here, it takes the property and its value from create-weblogic-domain-inputs.yaml
+adminServerName= admin-server
+domainName= base_domain
+domainUID= domain1
+startupControl= AUTO
```

Comment:
Remove default values? e.g. startupControl = AUTO
(this will make it easier to migrate your tests to the new lifecycle and WDT stuff)